### ISSUE TITLE
Importing CSV file from production TSDB " invalid INSERT on the root table of hypertable 'facts' "

### ISSUE THREAD

**Issue by samiam9297** [timescale/timescaledb-parallel-copy#38]
On my production TSDB, I use the following to export to CSV
`psql -d ccl -c "\COPY (SELECT * FROM facts BETWEEN '2020-04-27 14:00:00' AND '2020-04-27 17:00:01') TO dataset.csv DELIMITER ',' CSV"`

Which gives me output in the `CSV` similar to this (no headers):
`1014074658,Unit_C_Power,2020-04-27 17:00:00.622106,0`

I then dump the schema of the production database and import to my testing machine with a different DB name. Here is what the schema for table `facts` looks like:
![image](https://user-images.githubusercontent.com/61100078/80606204-08c64280-8a02-11ea-9d9f-e8d105dbe99c.png)

Then, I use the following to import the CSV generated earlier:
`timescaledb-parallel-copy --db-name cclapr --table facts --file "C:\dataset.csv" --workers 2 --reporting-period 30s`

And I am returned this error:
`panic: pq: invalid INSERT on the root table of hypertable "facts"`

Additionally I tried `psql` CSV import.
```
psql -U postgres -d cclapr -c "\COPY facts FROM C:\dataset.csv CSV"
ERROR:  invalid INSERT on the root table of hypertable "facts"
HINT:  Make sure the TimescaleDB extension has been preloaded.
CONTEXT:  COPY facts, line 1: "1014074658,Unit_C_Power,2020-04-27 17:00:00.622106,0"
```

I have confirmed `timescaledb` extension is enabled using the following in the docs:
```
\c cclapr
CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
```

Any guidance would be appreciated.

**Comment by samiam9297** [comment]
The solution was more obvious after I spelled this all out. I had to drop `ts_insert_blocker` to get this working.

Could this be included in the docs? I struggled for quite some time with this, it may prove helpful to others.

**Comment by RobAtticus** [comment]
That does not sound like the correct solution -- I believe that trigger is needed to prevent data from being stored in what should be the parent table of the chunk hierarchy that TimescaleDB uses.

From the error it looks like you did not run `timescaledb-tune` or update your postgresql.conf to include `timescaledb` under the `shared_preload_libraries` (and then restarted). Without doing so, then the TimescaleDB extension will not actually function correctly.

**Comment by samiam9297** [comment]
@RobAtticus I was able to query the data after doing the above. However, I did see adding `timescaledb` to `shared_preload_libraries` as a solution in other issues and confirmed it was already in `postgresql.conf` prior to my solution in comment [1], from the TSDB installation process which includes `timescaledb-tune`.

```
shared_preload_libraries = 'timescaledb'	# (change requires restart)
```

I also did several restarts. I assume the above

[1] : https://github.com/timescale/timescaledb-parallel-copy/issues/38#issuecomment-621246454
